charts/clamav/templates/clamav-mirror-configmap.yaml

```diff
@@ -23,15 +23,27 @@ data:
     )
     index-file.names = ( "index.htm" )
     server.modules += ( "mod_accesslog" )
-    accesslog.filename  = "/dev/stderr" 
+    accesslog.filename  = "/dev/stderr"
+    $HTTP["url"] == "/server-status" { 
+      accesslog.filename = "/dev/null"
+    }
+    $HTTP["useragent"] =~ "kube-probe" {
+      accesslog.filename = "/dev/null"
+    }
 
-  lightttpdmirror.conf: |
+  lighttpdmirror.conf: |
```

Comment:
typo in the helm chart meant this file wasn't being used

Reply:
I assume this should be `lighthttpmirror` ?

Reply:
Oh, I see the package is `lig HTTP` ignore me.

Reply:
Agreed, the name kind of hurts my brain

Reply:
Agreed, the name kind of hurts my brain